kafka-ui-api/src/main/java/com/provectus/kafka/ui/emitter/BackwardRecordEmitter.java

```diff
@@ -47,8 +48,9 @@ public void accept(FluxSink<TopicMessageEvent> sink) {
                )
       ) {
         sendPhase(sink, "Created consumer");
+
         final Map<TopicPartition, Long> partitionsOffsets =
-            offsetsSeek.getPartitionsOffsets(consumer);
+            new TreeMap<>(offsetsSeek.getPartitionsOffsets(consumer));
```

Comment:
TopicPartition is not comparable, it will fail,
suggestion 
```
        SortedMap<TopicPartition, Long> partitionsOffsets =
            new TreeMap<>(Comparator.comparingInt(TopicPartition::partition));
        partitionsOffsets.putAll(offsetsSeek.getPartitionsOffsets(consumer));
```